docs/src/snforge-advanced-features/storage-cheatcodes.md

```diff
@@ -160,4 +161,41 @@ fn store_in_complex_mapping() {
 > The `load` cheatcode will return zeros for memory you haven't written into yet (it is a default storage value for Starknet contracts' storage).
 
 
+## Example with `storage_address_from_base`
+
+This example use `storage_address_from_base` with `address` function of the [storage variable](https://book.cairo-lang.org/ch14-01-contract-storage.html#addresses-of-storage-variables).
+
+```rust
+#[starknet::contract]
+mod Contract {
+    #[storage]
+    struct Storage {
+        votes: LegacyMap::<(u8, u32), u32>,
```

Comment:
I would just say it's a `map` instead - it's not an important information that it's a vote here

Reply:
I have renamed `votes` to `map`